mozilla-mobile/fenix/app/src/main/res/values/strings.xml

```diff
@@ -1115,4 +1145,57 @@
     <string name="search_edit_custom_engine_success_message">Saved %s</string>
     <!-- Text shown when a user successfully deletes a custom search engine -->
     <string name="search_delete_search_engine_success_message">Deleted %s</string>
+
+    <!-- Description text for the Firefox brand logo -->
+    <string name="firefox_logo_description">Firefox logo</string>
+    <!-- Title text shown for the migration screen to the new browser -->
+    <string name="migration_title">Welcome to an all-new Firefox</string>
+    <!-- Description text followed by a list of things migrating (e.g. Bookmarks, History) -->
+    <string name="migration_description">A completely redesigned browser awaits, with improved performance and features to help you do more online.\n\nPlease wait while we update Firefox with your</string>
+    <!-- Text on the disabled button while in progress -->
+    <string name="migration_updating_app_button_text">Updating %s…</string>
+    <!-- Text on the enabled button -->
+    <string name="migration_update_app_button">Start %s</string>
```

Comment:
Please always explain what placeholder will be replaced with in comments

Reply:
@Delphine I had added "Placeholder replaced with app name", should I be more specific with examples of app names?

Reply:
@flodolo if you could help clarify this question as well, I can have it fixed with the other change too.

Reply:
This is already fixed at this point. The initial request was to explain what replaces `%s`, and now there's `Placeholder replaced with app name`.